client/app/scripts/liveblog-edit/module.js

```diff
@@ -461,8 +461,14 @@ define([
         function canPublishAPost(blog) {
             return privileges.userHasPrivileges({'publish_post': 1});
         }
-        function isUserOwner(archive) {
-            if ($rootScope.currentUser._id === archive.original_creator || $rootScope.currentUser.user_type === 'administrator') {
+        function accessToSettings(archive) {
+            //if no role assigned, assume no restriction from the role POV
+            var rolePermission = true;
+            if ($rootScope.currentUser.role) {
+                rolePermission = rolePermission && privileges.userHasPrivileges({'blogs': 1});
+            }
+            if (($rootScope.currentUser._id === archive.original_creator && rolePermission) ||
```

Comment:
An administrator has by default all the privileges. You don't have to check if he is administrator if you already check the privileges